.github/workflows/pr.yaml

```diff
@@ -28,6 +28,11 @@ jobs:
         with:
           cmakeVersion: 3.22.1 # ubuntu 22.04
           ninjaVersion: 1.11.1
+      - name: Verify Messages
+        shell: pwsh
+        run: |
+          cmake --preset windows-ci
+          cmake --build --preset windows-ci --target verify-messages -- -k0
```

Comment:
It seems like this should reuse the configure from line 39/44 rather than configuring again?